src/Helpers/constants.js

```diff
@@ -18,10 +18,10 @@ import moment from 'moment';
 import React from 'react';
 /* eslint react/prop-types: 0 */
 
-export const DEFAULT_PAGE_SIZE = 25;
+export const DEFAULT_PAGE_SIZE = 20;
 export const PER_PAGE_OPTIONS = [
```

Comment:
since we're using the default options now we can remove this constant.
 
From PF documentation `perPageOptions ` https://www.patternfly.org/v4/documentation/react/components/pagination#props has the same default values